patroni/ha.py

```diff
@@ -462,6 +462,8 @@ def touch_member(self) -> bool:
                     timeline, wal_position, pg_control_timeline = self.state_handler.timeline_wal_position()
                     data['xlog_location'] = self._last_wal_lsn = wal_position
                     if not timeline:  # running as a standby
+                        data['received_location'], data['replayed_location'] =\
```

Comment:
Should we better call them `receive_lsn` and `replay_lsn`?
After all, they were renamed in PG10, and unlike REST API we don't need to be backward compatible here.

Reply:
Also, we can also rename respective properties/methods in Postgresql and Member classes.